### ISSUE TITLE
add csv import format 

### ISSUE THREAD

**Issue by Bobcadence** [X-Ryl669/BwareArea#7]
It could be cool to add more import format like csv or other format.
May be suggest some reliable data provider, could help some newbie

By the way, a French radar database could be find here:
https://www.data.gouv.fr/fr/datasets/radars-automatiques/

**Comment by X-Ryl669** [comment]
I've tried to import CSV first (and the iGo format I'm supporting is actually a CSV). The issue is that CSV is not standard, there are as many CSV format as there are different software.
Some have headers (so I could probably "find a logic") but some haven't (so the meaning of the data is well... unknown). The approach in PinPOI is to ask the user to specify what the columns are before importing, but honestly, I think this is too complex for anyone without good GPS/GIS knowledge (and I can't use PinPOI code since it's GPL).

The iGO format is the more complete I could find, so I've used it (it has both the speed, direction and type as columns) while other found formats split each type in different CSV files and some even split each speed in different CSV files.

For a french & european database, you can visit Lufop.net and download their speedcam.txt file from iGO page. I guess I'm not supposed to store the file here, so you'll have to find it by yourself but it's quite straightforward.

**Comment by Hoiss** [comment]
Nice work and thanks for the hint with Lufop.net!
I downloaded the file for europe which includes 27033 POIs. After importing this file to Bware it shows only 19988 POIs. This seems to be a **bug with negative X values** which are valid for parts of france and spain!
You also have to be careful with the filename, it was accepted when it was named speedcam.txt like it is originally.

**Comment by X-Ryl669** [comment]
Right, I was wondering why there was a difference in the number of POI. I'll take a look.

**Comment by X-Ryl669** [comment]
Ok, I've made a new release with the bug fixed. Cheers!
